camel-openmrs-fhir/src/test/java/org/openmrs/eip/fhir/spring/OpenmrsFhirConfigurationTest.java

```diff
@@ -29,7 +29,8 @@
 @ContextConfiguration(classes = OpenmrsFhirConfiguration.class)
 @TestPropertySource(properties = { "eip.fhir.resources=Encounter", "openmrs.eip.log.level=DEBUG",
         "eip.fhir.serverUrl=http://localhost:8080/openmrs/ws/fhir2/R4", "eip.fhir.username=admin",
-        "eip.fhir.password=password" })
+        "eip.fhir.password=password", "openmrs.baseUrl=http://localhost:8080/openmrs", "openmrs.username=admin",
+        "openmrs.password=password" })
```

Comment:
Is there a misconfiguration here? Are the newly added properties used by this class?

Reply:
Yeah I think this test needs all the environment variables to be set otherwise the configuration test fails.